integrationTests/relayers/slowTests/testdata/contracts/mvx/helper-contract.abi.json

```diff
@@ -0,0 +1,86 @@
+{
```

Comment:
where is the code for this contract? Can you please link this PR with the PR on the rust contracts repo (if you have there the contract code)?